src/App.js

```diff
@@ -53,11 +56,17 @@ function App() {
   }
 
   function handlePlay() {
-    !isPlaying && toggle()
+    if (!isPlaying) {
+      toggle()
+    }
     if (isAnswerVisible && !isPlaying) {
       setIsAnswerVisible(false)
     }
   }
+
+  function changeVolume(event) {
+    volume(event.target.value)
+  }
```

Comment:
I don't like that a function `changeVolume` has a parameter `event`. It should just get a value.